tests/extensions/zip/fixtures/obj_list.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Here is an example of the `filename` for each `file` in `ZipFile().filelist`. It is confusing since the `filename` is the full path for a file or a folder. In the code in `render.py`, I try to make this less confusing by using `obj` for var names.